purchases/src/defaults/kotlin/com/revenuecat/purchases/listenerConversions.kt

```diff
@@ -205,6 +219,23 @@ fun Purchases.syncAttributesAndOfferingsIfNeededWith(
     syncAttributesAndOfferingsIfNeeded(syncAttributesAndOfferingsListener(onSuccess, onError))
 }
 
+/**
+ * Note: This method only works for the Amazon Appstore. There is no Google equivalent at this time.
+ * Calling from a Google-configured app will always return AmazonLWAConsentStatus.UNAVAILABLE.
+ *
+ * Get the Login with Amazon consent status for the current user. Used to implement Quick Subscribe.
+ *
+ * @param [onSuccess] Called when the consent status was successfully fetched.
+ * @param [onError] Called when there was an error fetching the consent status.
```

Comment:
Nitpick, but we should order these in the order of the params, so we should switch these
```suggestion
 * @param [onError] Called when there was an error fetching the consent status.
 * @param [onSuccess] Called when the consent status was successfully fetched.
```

Reply:
Eneded up changing the order of the parameters instead of the documentation.

Reply:
Hmm I think our usual convention is to put the `onError` first and `onSuccess` after, so I would actually rollback that change and change the documentation, so it's consistent.

Reply:
The order of the parameters was my reasoning, and it the reason for having the `onSuccess` at the end is so the `onSuccess` is the one that gets the lambda conversion, which I think it's cleaner.

```
fun Purchases.getAmazonLWAConsentStatus(::onError) {
	// do something
}
```

Better than:

```
fun Purchases.getAmazonLWAConsentStatus(onSuccess = { 
 // do something on success
} ) {
	// do something on error
}
```

Reply:
Good catch I didn't notice the naming or the convention in the other methods, fixed!